meteor/lib/collections/RundownPlaylists.ts

```diff
@@ -35,51 +35,70 @@ export type RundownPlaylist = DBRundownPlaylist
  * These used to reside on the Rundown class
  */
 export class RundownPlaylistCollectionUtil {
-	/** Returns all Rundowns in the RundownPlaylist */
-	static getRundowns(
-		playlist: Pick<RundownPlaylist, '_id'>,
+	static getRundownsOrdered(
+		playlist: Pick<RundownPlaylist, '_id' | 'rundownIdsInOrder'>,
 		selector?: MongoQuery<Rundown>,
 		options?: FindOptions<Rundown>
 	): Rundown[] {
-		return Rundowns.find(
+		const allRundowns = Rundowns.find(
 			{
 				playlistId: playlist._id,
 				...selector,
 			},
 			{
 				sort: {
-					_rank: 1,
 					_id: 1,
 				},
 				...options,
 			}
 		).fetch()
+
+		const rundownsMap = normalizeArrayToMap(allRundowns, '_id')
+
+		const sortedIds = sortRundownIDsInPlaylist(playlist.rundownIdsInOrder, Array.from(rundownsMap.keys()))
+
+		return _.compact(sortedIds.map((id) => rundownsMap.get(id)))
 	}
-	/** Returns an array with the id:s of all Rundowns in the RundownPlaylist */
-	static getRundownIDs(
+
+	static getRundownsUnordered(
 		playlist: Pick<RundownPlaylist, '_id'>,
 		selector?: MongoQuery<Rundown>,
 		options?: FindOptions<Rundown>
-	): RundownId[] {
-		return RundownPlaylistCollectionUtil.getRundowns(playlist, selector, {
-			sort: {
-				_rank: 1,
-				_id: 1,
-			},
-			fields: {
-				_rank: 1,
-				_id: 1,
+	): Rundown[] {
+		return Rundowns.find(
+			{
+				playlistId: playlist._id,
+				...selector,
 			},
-			...options,
-		}).map((i) => i._id)
+			{
+				sort: {
+					_id: 1,
+				},
+				...options,
+			}
+		).fetch()
 	}
-	static getRundownUnorderedIDs(playlist: Pick<RundownPlaylist, '_id'>, selector?: MongoQuery<Rundown>): RundownId[] {
-		return RundownPlaylistCollectionUtil.getRundowns(playlist, selector, {
-			fields: {
-				_id: 1,
+	/** Returns an array with the id:s of all Rundowns in the RundownPlaylist */
+	static getRundownOrderedIDs(playlist: Pick<RundownPlaylist, '_id' | 'rundownIdsInOrder'>): RundownId[] {
+		const allIds = RundownPlaylistCollectionUtil.getRundownUnorderedIDs(playlist)
+
+		return sortRundownIDsInPlaylist(playlist.rundownIdsInOrder, allIds)
+	}
+	static getRundownUnorderedIDs(playlist: Pick<RundownPlaylist, '_id'>): RundownId[] {
```

Comment:
This method could use a description, perhaps also with a hint of when to use it compared to `getRundownOrderedIDs`.